tests/test_model_functions.py

```diff
@@ -84,6 +84,24 @@ def test_get_derived_quantities(self, basic_model, simple_ode_model):
             '_deriv_on_rhs2b$sv1_rate]'
         )
 
+    def test_get_display_name(self):
```

Comment:
I'd like lower-level tests here. So check not all states, but individual variables covering the cases:
* Has an oxmeta-annotation
* Has an annotation in another ontology
* Has no annotation but does have a cmeta:id
* Has no cmeta:id

Reply:
Ok I checked 4 individual variables for these for cases in test_simple_odes.cellml instead